artiq/firmware/libboard_artiq/hmc830_7043.rs

```diff
@@ -224,10 +225,17 @@ pub mod hmc7043 {
         Ok(())
     }
 
-    pub fn shutdown() -> Result<(), &'static str> {
+    pub fn enable() -> Result<(), &'static str> {
+        info!("enabling hmc7043");
+
+        unsafe {
+        csr::crg::hmc7043_rst_write(0);
```

Comment:
Indent

Reply:
And there is no time for the HMC7043 to wreak havoc between the release of its reset line and the SPI write that mutes it?